react/app-admin/components/Statistics/SearchTagsChart.tsx

```diff
@@ -3,164 +3,158 @@ import { Form, Select } from 'semantic-ui-react';
 import { serializeSearchTagsStatsFromAPI } from 'tmw-admin/utils/api-serialize';
 import { ajaxPost } from 'tmw-common/utils/ajax';
 import { getApiDateFormat } from 'tmw-common/utils/date';
-import {Chart} from 'chart.js';
+import { Chart } from 'chart.js';
 import { convertToSelectOptions, InputSelectOption } from 'tmw-admin/utils/select-options';
-import {SearchTagStat} from 'tmw-admin/constants/app-types'
-
+import { SearchTagStat } from 'tmw-admin/constants/app-types';
 
 export const SearchTagsChart: React.FunctionComponent = () => {
     const [parentTagOptions, setParentTagOptions] = React.useState<InputSelectOption[]>([]);
-    const [selectedTagOption, setselectedTagOption] = React.useState<string>('parent_slug')
+    const [selectedTagOption, setselectedTagOption] = React.useState<string>('parent_id');
     const [errorMessage, setErrorMessage] = React.useState<string>('');
     const [searchTagsStats, setSearchTagsStats] = React.useState<SearchTagStat[]>([]);
     const [chart, setChart] = React.useState<Chart>();
 
-    const getParentTagOptions = () : InputSelectOption[] => {
-        let options :InputSelectOption[] = [
+    const getParentTagOptions = (): InputSelectOption[] => {
+        const options: InputSelectOption[] = [
             {
-                key: "parent_slug",
-                value: "parent_slug",
-                text: "Tags principaux",
-            }
+                key: 'parent_id',
+                value: 'parent_id',
+                text: 'Tags principaux',
+            },
         ];
-        searchTagsStats.map(t => options.push({
-            key: t.slug,
-            value: t.slug,
-            text: t.name,
-        }));
+        searchTagsStats.filter(t => t.is_parent)
+            .map(t =>
```

Comment:
You can use the `convertToSelectOptions` in `utils/select-options` to convert tags to select options 🙂

Reply:
Done !